feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/CameraControlsOverlay.kt

```diff
@@ -294,7 +294,11 @@ private fun ControlsBottom(
         ) {
             Column(horizontalAlignment = Alignment.CenterHorizontally) {
                 if (showZoomLevel) {
-                    ZoomScaleText(zoomLevel)
+                    zoomRatios[previewUiState.currentCameraSettings.cameraLensFacing]?.let {
```

Comment:
We shouldn't introduce new usages of `currentCameraSettings` into the UI. The zoom ratio to show should be determined at the ViewModel level. You shouldn't need to pipe through a Map<LensFacing, Float>. If you want to show multiple zoom ratios in the UI, that kind of state should be encapsulated by something like a `ZoomRatioUiState`.